pkg/diff2/ordering.go

```diff
@@ -0,0 +1,18 @@
+package diff2
+
+import (
+	"log"
+
+	"github.com/StackExchange/dnscontrol/v4/pkg/dnsgraph"
+	"github.com/StackExchange/dnscontrol/v4/pkg/dnssort"
+)
+
+func orderByDependencies(changes ChangeList) ChangeList {
```

Comment:
Let's put this feature (in all  behind a flag at least temporarily.  The flag should default to on, but `--disableordering` would make this function `return changes` unmodified.

See `dnscontrol/pkg/bindserial/main.go`  for how to add a global flag that is temporary.

I'm not saying that the code is bad or anything... just that new features like this should be easy to disable until we've had them in production for a while.

Tom

Reply:
I understand, the fact that you want to merge this and have it on by default is trust enough ;) 

I added the flag and added an informational message (just like with diff2) as well.

Reply:
Looks good!  Thanks for understanding!